sunpy/map/tests/test_mapbase.py

```diff
@@ -231,6 +233,62 @@ def test_data_to_pixel(generic_map):
     test_pixel = generic_map.data_to_pixel(*generic_map.reference_coordinate, origin=1)
     assert_quantity_allclose(test_pixel, generic_map.reference_pixel)
 
+def test_default_shift():
+    """Test that the default shift is zero"""
+    data = np.ones([6,6], dtype=np.float64)
+    header = {'CRVAL1': 0,
+              'CRVAL2': 0,
+              'CRPIX1': 5,
+              'CRPIX2': 5,
+              'CDELT1': 10,
+              'CDELT2': 9,
+              'CD1_1': 0,
+              'CD1_2': -9,
+              'CD2_1': 10,
+              'CD2_2': 0,
+              'NAXIS1': 6,
+              'NAXIS2': 6}
+    cd_map = sunpy.map.Map((data, header))
+    assert cd_map.shifted_value.x.value == 0
+    assert cd_map.shifted_value.y.value == 0
+
+def test_shift_applied(generic_map):
```

Comment:
with respect to @dpshelio comment in the main PR, you could parametrise the x_shift and y_shift here.